docs/D-Modularity/functions-arrays-and-structs.md

```diff
@@ -473,7 +485,7 @@ The C compiler performs member-by-member copying automatically whenever we:
 * assign an object to an existing object
 * initialize a new object using an existing object
 * return an object by value
-* Pass By Address
+* pass by address
```

Comment:
I think the first letter in the list should be capitalized as well

Reply:
Since all other list items are lowercase, capitalizing this would not be consistent.